books/free-programming-books-es.md

```diff
@@ -291,12 +298,18 @@
 * [Manual de estudio introductorio al lenguaje PHP procedural](https://web.archive.org/web/20140209203630/http://www.cursosdeprogramacionadistancia.com/static/pdf/material-sin-personalizar-php.pdf) - Eugenia Bahit (PDF)
 * [PHP y Programación orientada a objetos](https://styde.net/php-y-programacion-orientada-a-objetos/) - Duilio Palacios (HTML)
 * [POO y MVC en PHP](https://bibliotecafacet.com.ar/wp-content/uploads/2014/12/eugeniabahitpooymvcenphp.pdf) - Eugenia Bahit (PDF)
+* [Prepárate para una entrevista técnica PHP](https://programacionymas.com/blog/php-preguntas-entrevista-trabajo) - Blog de "Programacion y más" (HTML)
```

Comment:
Blog Post?

Reply:
I believe this questions and answers compilation counts as a book. I could share a PDF version of this, but there are other resources shared as HTML too.
Please let me know.

Reply:
Please, should be removed